flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/operators/rank/window/WindowRankOperatorBuilder.java

```diff
@@ -152,6 +152,7 @@ public WindowRankOperatorBuilder windowEndIndex(int windowEndIndex) {
                         outputRankNumber,
                         windowEndIndex,
                         shiftTimeZone);
-        return new WindowAggOperator<>(windowProcessor);
+        // Processing time Window TopN is not supported yet.
+        return new WindowAggOperator<>(windowProcessor, true);
```

Comment:
Is there any attribute representing eventtime? If yes, we can use a judgement expression instead of a literal `true` value.